persistence/src/commonMain/db_user/com/wire/kalium/persistence/Notification.sq

```diff
@@ -16,12 +16,12 @@ SELECT
     Conversation.type AS conversationType,
     Conversation.degraded_conversation_notified AS degradedConversationNotified,
     Conversation.legal_hold_status AS legalHoldStatus,
-    LegalHoldNotified.legal_hold_status_change_notified AS legalHoldStatusChangeNotified
+    IFNULL(LegalHoldNotified.legal_hold_status_change_notified, 1) == 1 AS legalHoldStatusChangeNotified
```

Comment:
Here I assumed the default value from the table, but please @borichellow that has more context, correct me if otherwise :D